### ISSUE TITLE
Very Large Default Learning Rate for Adadelta?

### ISSUE THREAD

**Issue by mossaab0** [fchollet/keras#3889]
I'm wondering how the default learning rate for adadelta was set. `lr=1.0` is much larger than the corresponding value for the other optimizers (`0.001<=lr<=0.01`). Isn't this strange?

(I know I can set any value I want. But I want to use the default options as much as I can)

**Comment by crowsonkb** [comment]
Adadelta was meant to not require manual learning rate tuning, so [the Adadelta paper](https://arxiv.org/abs/1212.5701) didn't even include a learning rate parameter. Most real-world implementations include a learning rate parameter but let it default to 1, which leaves the automatically-determined learning rates alone.

**Comment by mossaab0** [comment]
So should we set `lr=1.0` by default for the other optimizers?

**Comment by crowsonkb** [comment]
No, the other optimizers have reasonable defaults for them (and in the case of Adam at least the defaults are the ones recommended in [the paper](http://arxiv.org/abs/1412.6980)).

**Comment by hav4ik** [comment]
I believe [this StackOverflow thread](https://stackoverflow.com/questions/56730888/what-is-the-learning-rate-parameter-in-adadelta-optimiser-for-in-keras) answers this question.
